src/wallet/wallet.h

```diff
@@ -732,11 +723,9 @@ class CWallet final : public WalletStorage, private interfaces::Chain::Notificat
     {
         // Should not have slots connected at this point.
         assert(NotifyUnload.empty());
-        delete encrypted_batch;
-        encrypted_batch = nullptr;
     }
 
-    bool IsCrypted() const { return fUseCrypto; }
+    bool IsCrypted() const;
```

Comment:
In commit "Refactor: Move encryption code between KeyMan and Wallet" (d96925bde5c103693e44787503dc3e45d1248da8)

Future cleanup could delete this method and just have callers call `HasEncryptionKeys()` directly instead.